tests/fv3_conf/fv3_qsub.IN_wcoss2

```diff
@@ -15,8 +15,8 @@ echo -n " $( date +%s )," >  job_timestamp.txt
 cd $PBS_O_WORKDIR
 
 set +x
-module use $( pwd -P )
-module load modules.fv3
+module use modulefiles
```

Comment:
I am curious, do we have modulefiles in the working directory?

Reply:
Never mind.

Reply:
One thing I noticed from Cheyenne when admin upgraded lua last year... new lua prefers only module files in single directory. If mixed files exist in same directory, lua may complain.